rpc-rs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmbus-rpc"
-version = "0.9.3"
+version = "0.10.0"
```

Comment:
Were there any other changes that we needed to bring in before we bump?

Reply:
We don't have to publish immediately if there are other things we're waiting on. This change is not super-urgent

Reply:
I wouldn't be surprised if there is an update to async-nats in a couple weeks that we'll want to push ..

Reply:
We should take a look and see if we can get any async-nats fixes before we publish. It is true that once we merge this PR we'd have to cherry-pick if we needed to release a patch version to `0.9`